packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewer.js

```diff
@@ -83,30 +137,93 @@ class LightCurveViewer extends Component {
         .call(setPointCoords)
     }
   }
+  
+  getCurrentTransform () {
+    return (d3.event && d3.event.transform)
+      || d3.zoomTransform(this.d3interfaceLayer.node())
+      || d3.zoomIdentity
+  }
 
+  /*
+  Initialises the D3 scatterplot chart.
+  The chart is divided into multiple layers (both functional and decorative).
+  IMPORTANT: note the order these layers are added.
```

Comment:
This comment could be made more helpful by adding a note to the effect that layers are added in z-index order, lowest first